specification/2.0/schema/material.schema.json

```diff
@@ -10,7 +10,7 @@
         "extras": { },
         "pbrMetallicRoughness": {
             "allOf": [ { "$ref": "material.pbrMetallicRoughness.schema.json" } ],
-            "description": "A set of parameter values that are used to define the metallic-roughness material model from Physically-Based Rendering (PBR) methodology. When undefined, all the default values of `pbrMetallicRoughness` **MUST* apply."
```

Comment:
This wasn't introduced in this PR, but there's a `**MUST*` here, missing an asterisk.